<launch>
<!-- This launch file is for policies trained with isaac lab and the model is exported to onnx -->
  <arg name="cmd_topic" default="/control_agent/cmd_drive"/>
  <arg name="odom_topic" default="/local_odom"/>
  <arg name="goal_topic" default="/move_base_simple/goal"/>

  <node pkg="kingfisher_rl" type="rans_agent.py" name="rl_agent" output="screen">
    <remap from="~goal" to="$(arg goal_topic)"/>
    <remap from="~odom" to="$(arg odom_topic)"/>
    <remap from="~cmd_drive" to="$(arg cmd_topic)"/>
  </node>

</launch>
